views/choose_song.erb

```diff
@@ -0,0 +1,20 @@
+<% if @hits.count == 1 %>
+<h2> Please confirm this is the song you wish to add:</h2>
+<% else %>
+<h2><%= @hits.count %> songs fit that criteria</h2>
+<h2> Please confirm the song you wish to add:</h2>
+<form align="center" action="/no_song" method="POST">
+  <input type="submit" value="Song not on list">
+</form>
+<% end %>
+<%  @hits.each do |song|   %>
```

Comment:
It'd be good to indent here so that you can tell at a glance what happens in this `each` loop